### DISCUSSION TITLE
Help! Importing an image without a background.

### DISCUSSION THREAD

**Discussion by Artemikhaylovsky** [keiwando/toonsquid#18] [Q&A] [answered]
Hello, how to import PNG, transparently, without background?

When importing PNG, a white background appears.

**Comment by keiwando** [comment]
Hi! 
Did you already disable the visibility of the background layer in the timeline? https://toonsquid.com/handbook/layers/layer_list/#visibility

**Reply by keiwando** [reply]
Could you maybe upload the PNG here or send it to me via contact@toonsquid.com so that I can try it out?
I just double checked with two different PNGs and they were imported correctly with transparencies (both when importing from the Files app and from the Photos app).

**Reply by Artemikhaylovsky** [reply]
![40F0D529-BD63-4C6F-9EA2-BD17CCD0D458](https://user-images.githubusercontent.com/115465544/194883782-996ba913-f5ef-4212-b533-12611df2c134.png)

**Comment by keiwando** [comment]
Thank you!

I just tried it with that image on three different iPads and iOS versions and it worked in all of them:

![IMG_B064FC372C6E-1](https://user-images.githubusercontent.com/12989570/194889343-ed686888-acc8-43b5-92f8-5982f3a7917e.jpeg)

Could you please tell me your iPad model and the iOS version that you are running?

Also, just because I've noticed it in your screenrecording: Do you actually see those three multitasking dots at the top in the app or is that only added automatically in the recording? They should not show up in ToonSquid

**Reply by Artemikhaylovsky** [reply]
I have an iPad Pro 11" 3rd generation
Software version - 15.1

I'm from Russia

**Reply by Artemikhaylovsky** [reply]
I have these three dots always present in any applications for simultaneous display of multiple applications

**Reply by keiwando** [reply]
Thanks for the added info. One of the iPads I'm trying this on is the iPad Pro 11" 2018 running iOS 15.7 and it works there. I'm seeing various reports in the Apple forums of people having issues with transparent PNGs after updating to iOS 15, but some of them are saying that the transparency is gone even in the Photos app, which is not the case for you. It would still be good if you could try to add the images from the Files app instead of the Photos app and see if that makes any difference.

The next possibility is very unlikely to be the cause but let's still make sure: can you double tap the image in ToonSquid (which should open the symbol that the image has been imported into) and there make sure that the app has properly disabled the background layer in that symbol as well.

If neither of those attempts make a difference, I would next suggest to try an updated version of iOS 15, since it could be a bug in there.

**Reply by Artemikhaylovsky** [reply]
I just upgraded iOS to 15.7 and now everything is working correctly. Thanks for the support! 

Finally, there is a decent animation application for a nice payment!

**Reply by keiwando** [reply]
I'm very glad to hear that. Thank you for the kind words!

**Comment by keiwando** [comment, marked as answer]
> I just upgraded iOS to 15.7 and now everything is working correctly.
https://github.com/keiwando/toonsquid/discussions/18#discussioncomment-3840548

^ Just for reference in case somebody else comes across this with the same issue
